code-challenges.js

```diff
@@ -1,69 +1,127 @@
 // ASSESSMENT 1: Coding Practical Questions
 
-// To run file: navigate to the correct directory in terminal, $ node code-challenges.js
+// 1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
+    
+    const temperature1 = 42
+        // Expected output: "42 is below boiling point"
 
-// Pseudo coding your process is required.
+    const temperature2 = 350
+        // Expected output: "350 is above boiling point"
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in two strings and determines which of the two strings has more characters. Use the two sets of test variables provided.
+    const temperature3 = 212
+        // Expected output: "212 is at boiling point"
 
-// Pseudo code:
+// Pseudo code: 
+    // The variable temp represents the number inputted.
+    // The function hotOrNot will determine if the variable temp is below, above, or at boiling point.
+    // Only the variable temperature3 is necessary since it would be redundant to use the other variables.
+    // However, since there are many possible numbers inputted, using conditionals such as if, else if, and else will be used.
+    // The function hotOrNot will need to return a string that needs to display the following:
+        // If the temp input is less than temperature3, it will need to show that it is below boiling point.
+        // If the temp input is equal to temperature3, it will need to show that it is at boiling point.
+        // If the temp input is greater than temperature3, it will need to show that it is at boiling point. 
 
-// Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
+const hotOrNot = (temp) => {
+    if (temp < temperature3) {
+        return (temp + " is below boiling point")
+    } else if (temp > temperature3) 
+        return (temp + " is above boiling point")
+        else (temp = temperature3)
+        return (temp + " is at boiling point")
+    }
```

Comment:
Don't forget you can use string interpolation instead of concatenation for your return! Also, I see that you used temperature3 as the boiling point temp, but since it's a given variable, avoid using given variables in your function logic as best practice! Love your pseudo code!